nodebuilder/da/service.go

```diff
@@ -126,6 +152,36 @@ func (s *Service) Submit(
 	return ids, nil
 }
 
+func parseOptions(options []byte) ([]state.ConfigOption, error) {
+	var opts []state.ConfigOption
+
+	parsedOpts := struct {
```

Comment:
doesn't this need json tags?

Reply:
No, we're using defaults, so it's just the name. 
See: https://go.dev/play/p/I4925joE12fhttps://go.dev/play/p/I4925joE12f